frontend/src/components/LoginForm/LoginForm.type.ts

```diff
@@ -0,0 +1,12 @@
+import { FormInstance } from 'antd';
+import { LoginForm } from '@types';
+
+interface Props {
+  initialState: LoginForm;
+  onFinish: (values: any) => void;
+  onFinishFailed: (values: any) => void;
+  form: FormInstance<any>;
+  isLoading: boolean;
+}
+
+export default Props;
```

Comment:
Como solo usamos una vez este typo, lo podemos mover a este archivo, para no meter ruido a otros:
```suggestion
import { FormInstance } from 'antd';

export interface LoginForm {
  email: string;
  password: string;
}

interface Props {
  initialState: LoginForm;
  onFinish: (values: any) => void;
  onFinishFailed: (values: any) => void;
  form: FormInstance<any>;
  isLoading: boolean;
}

export default Props;

```

Reply:
Igual hay que ver si se puede remover el `any` y reemplazarlo con `LoginForm`.

Reply:
Y también hay que cuidar que todos los archivos terminen en una línea en blanco, para seguir el estándar POSIX.